xray_vision/mask/manual_mask.py

```diff
@@ -78,27 +81,35 @@ def __init__(self, ax, image, cmap='gray'):
             drawing canvas. Its content does not affect the output.
         cmap : str, optional
             'gray' by default
+        max_memory : int, optional
+            The maximum number of frames to keep in the undo history
+            Defaults to 20.
 
         Attributes
         ----------
         mask : boolean array
             all "postive" regions are True, negative False
+
         label_array : integer array
             each contiguous region is labeled with an integer
-        label_by_stroke : integer array
```

Comment:
Eh, on second thought it was cute but probably not that useful, and I can see how it's tricky to fit into the new pixel-flipping feature.